backend/app/api/v1/commons/responses.py

```diff
@@ -0,0 +1,18 @@
+def response_200(example):
```

Comment:
Can we name the file mock/metadata/sample like mock_response.py/ api_metadata.py/ sample_responses.py?